wtrocki/ionic4-grapqhl-starter
# Subscriptions support

## Motivation

Add subscription for Create Update Delete.

<img width="366" alt="screenshot" src="https://user-images.githubusercontent.com/981838/49592141-45eee800-f968-11e8-8f6f-bc6ba852e55e.png">

## Verification 

- Follow README to launch server and mobile app
- Open 2 tabs.
- Make modifications (create,delete,update)
- Observe changes being made on other tab.
